autospec/abireport.py

```diff
@@ -219,8 +219,19 @@ def examine_abi(download_path):
 
 def examine_abi_host(download_path, results_dir):
     """Make use of the hostside abireport tool."""
+    rpms = set()
+    for item in os.listdir(results_dir):
+        if item.find("-extras-") > 0:
```

Comment:
Checking for `-extras-` will also capture rpm names like `kio-extras-lib-19.04.3-29.x86_64.rpm`. To make it more robust, I suggest to account for the package name in the rpm name (see `tarball.name` global variable).

Reply:
works now.